src/manager.rs

```diff
@@ -146,10 +151,9 @@ where
         })
     }
 
-    /// Tries to close the specified environment and delete all its files from disk.
-    /// Doesn't delete the folder used when opening the environment.
-    /// This will only work if there's no other users of this environment.
-    pub fn try_close_and_delete<'p, P>(&mut self, path: P) -> Result<()>
+    /// Tries to close the specified environment.
+    /// Returns an error when other other users of this environment still exist.
```

Comment:
This used to fail silently if other environments existed. Now it returns an error instead.